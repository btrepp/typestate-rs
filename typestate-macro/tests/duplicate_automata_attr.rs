use typestate::typestate;

#[typestate]
mod m {
    #[automata]
    #[automata]
    struct S {}
}

fn main() {}