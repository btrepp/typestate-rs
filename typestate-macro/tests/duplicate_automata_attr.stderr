error: Duplicate attribute.
 --> $DIR/duplicate_automata_attr.rs:6:5
  |
6 |     #[automata]
  |     ^^^^^^^^^^^
